vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/ConsumableResource.java

```diff
@@ -78,7 +78,7 @@ public JavaType typeFromId(DatabindContext context, String id) throws IOExceptio
    * @param workflowVersion the version of the workflow
    * @param vidarrId the identifier of the workflow run
    */
-  void release(String workflowName, String workflowVersion, String vidarrId);
+  void release(String workflowName, String workflowVersion, String vidarrId, boolean isLaunched, Optional<JsonNode> input);
```

Comment:
Please update the javadoc for this interface and the Plugin Developers guide

Reply:
Also, imo 'isLaunched' is a little confusing from a plugin developer implementing an interface perspective. Since the consumable resource interface is designed for the idea that runs would hold onto resources through running until run completion, release() is called when a job transitions to succeeded or failed*, ie, terminal states. They not only launched, they completed too. 

* or is throttled but that's not a terminal state

Reply:
Good point, changed to isComplete, so it'll be be false when throttled which is hopefully more intuitive

Reply:
Perhaps `requeueWorkflowRun` instead of `isLaunched`?

Reply:
`requeueWorkflowRun` is a bit prescriptive for what future resources implementing this interface will do with the information, isn't it? with `isComplete` the server neutrally tells the resource what happened and the resource can decide what to do with that information.